outputs.tf

```
@@ -1,14 +1,219 @@
+output "assessment_private_zone" {
+  value       = aws_route53_zone.assessment_private
+  description = "The private DNS zone for this assessment."
+}
+
+output "assessor_portal_instance_profile" {
+  value       = aws_iam_instance_profile.assessorportal
+  description = "The instance profile for the Assessor Portal instances."
+}
+
+output "assessor_portal_instances" {
+  value       = aws_instance.assessorportal
+  description = "The Assessor Portal instances."
+}
+
+output "assessorportal_security_group" {
```

Is there a reason this output is named differently from the previous two for the same group of resources?

```suggestion
output "assessor_portal_security_group" {
```

---

See commit b7a7c97.